DESCRIPTION

```diff
@@ -49,12 +49,13 @@ Imports:
   shinycssloaders,
   tidyselect,
   xtable,
-  yaml
+  yaml, 
+  testthat, 
```

Comment:
Pakken trenger ikke 'testthat' for å fungere, men trengs kun for testing under utvikling av pakken. Bør settes under 'Suggests:' slik det er gjort i ablanor. Ellers bør pakkene settes inn i alfabetisk rekkefølge (særlig når det er skrekkelig mange slik som i noric...)